source/application/PGM/PGMAppController.cpp

```diff
@@ -137,7 +141,9 @@ void PGMAppController::registerDBClasses()
 
 void PGMAppController::registerExporterOptions()
 {
-
+        AMExporterOptionXDIFormat *pgmXASExportOptions = PGM::buildXDIFormatExporterOption("PGMXASDefault", true);
```

Comment:
In some of the scan configurations on other beamlines, we make an additional call to build exporter options in the scan configuration constructor, pretty much just repeating these calls made here. Just wanted to check whether that was needed on PGM as well.